assets/css/planning.scss

```diff
@@ -169,3 +169,15 @@ $tableBoxSize: 40px;
     }
   }
 }
+
+@media screen and (min-width: 1001px) {
+  .planning-actions-container {
+    position: sticky;
+    top: 0;
+    z-index: 1;
+    width: 100vw;
```

Comment:
you must put a size to the buttons container, as this line will create very large buttons (you can have a look to the screenshot I've put on your issue)

Reply:
this width is not required for this PR, it's just to have the form actions container beeing not "floating" into the page but that's a cosmetic issue, the real need is to have the actions constantly accessible.

Reply:
Please have a look to the screenshot I've put in the issue: the table headers are hidden, and buttons are now too large

Reply:
Yes I've seen the screenshot the PR is just not up to date.

Reply:
The table headers are fixed, Chrome doesn't apply the shift on `top` value on `thead` it has to be set on `th` cells.
I have add a `.planning-actions-container-wrapper`div to keep a background on the actions and prevent them to be "floating" in the page without removing the bootstrap sizing on action buttons.